src/main/java/com/google/cloud/tools/managedcloudsdk/install/Extractor.java

```diff
@@ -81,4 +82,10 @@ ExtractorProvider getExtractorProvider() {
   private void cleanUp(final Path target) throws IOException {
     MoreFiles.deleteRecursively(target, RecursiveDeleteOption.ALLOW_INSECURE);
   }
+
+  static boolean isTargetInsideDestination(Path destination, Path target) throws IOException {
```

Comment:
Seems odd that arguments are in a different order from what's implied in the method name.

Reply:
Switched the order.